python/heterocl/devices.py

```diff
@@ -145,8 +159,9 @@ def __init__(self, types, vendor, model, **kwargs):
 
         for key, value in kwargs.items(): 
             self.config[key] = value
+
         # connect to ddr by default
```

Comment:
`ddr` => `DRAM`? BTW, where is `DDR` configed?

Reply:
For commercial devices, we have a lookup table to check the DRAM configuration of particular FPGA boards. For custom boards, there is an interface to do so